src/main/java/org/folio/repository/packages/PackageRepository.java

```diff
@@ -3,11 +3,10 @@
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
-import org.folio.holdingsiq.model.PackageByIdData;
 import org.folio.holdingsiq.model.PackageId;
 
 public interface PackageRepository {
-  CompletableFuture<Void> save(PackageByIdData packageData, String tenantId);
+  CompletableFuture<Void> save(DbPackage packageData, String tenantId);
```

Comment:
Can we please rename DbPackage to something else - probably packageByIdDataInDB?